kitchen.yml

```diff
@@ -4,27 +4,23 @@ driver:
 provisioner:
   name: chef_zero
   chef_license: accept
-  # We have tests that ensure our deprecated features still
-  # work so we can't turn this on globally.
-  # deprecations_as_errors: true
+  deprecations_as_errors: true
+  enforce_idempotency: true
+  multiple_converge: 2
+
```

Comment:
Please remove the extra space here.

Reply:
oof, looked for a space for a bit, and didn't realize you meant newline lol